---
bool success
sensor_msgs/Image image
float32 time